src/test/java/teammates/test/cases/action/InstructorFeedbackEditCopyActionTest.java

```diff
@@ -413,6 +376,58 @@ protected InstructorFeedbackEditCopyAction getAction(String... params) {
     @Override
     @Test
     protected void testAccessControl() throws Exception {
-        //TODO: implement this
+        String[] params = new String[]{
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, "First feedback session",
+                Const.ParamsNames.COURSE_ID, "idOfTypicalCourse1",
+                Const.ParamsNames.COPIED_FEEDBACK_SESSION_NAME, "Session with valid name",
+                Const.ParamsNames.COPIED_COURSES_ID, "idOfTypicalCourse2",
+                Const.ParamsNames.COPIED_COURSES_ID, "idOfSampleCourse-demo"
+        };
+
+        verifyUnaccessibleWithoutModifySessionPrivilege(params);
+
+        InstructorAttributes instructor = dataBundle.instructors.get("teammates.test.instructor2");
+        String instructorId = instructor.googleId;
+        FeedbackSessionAttributes fs = dataBundle.feedbackSessions.get("openSession");
+        CourseAttributes course = dataBundle.courses.get("course");
+
+        gaeSimulation.loginAsInstructor(instructorId);
+
+        ______TS("Failure case: copying from course with insufficient permission");
+        params = new String[] {
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, fs.getFeedbackSessionName(),
+                Const.ParamsNames.COURSE_ID, "FeedbackEditCopy.CS2107",
+                Const.ParamsNames.COPIED_FEEDBACK_SESSION_NAME, "valid name",
+                Const.ParamsNames.COPIED_COURSES_ID, course.getId()
+        };
+        InstructorFeedbackEditCopyAction a = getAction(params);
+        String expectedString;
```

Comment:
Declare `expectedString` at line 409 and line 428 instead so that the code looks cleaner.

Reply:
Will do 
Forgot to save file before pushing :p